crates/config/src/config.rs

```diff
@@ -86,9 +86,26 @@ pub struct ChallengerConfig {
     pub burn_lock: Script,
 }
 
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
+pub struct BlockCooldown {
+    pub withdrawals: usize,
+    pub txs: usize,
+}
+
+impl Default for BlockCooldown {
+    fn default() -> Self {
+        BlockCooldown {
+            withdrawals: 10, // drop 10% of withdrawals
```

Comment:
I think we can just try to drop half txs to simplify our logic.

Reply:
Fixed